pkg/utils/netlink_provider.go

```diff
@@ -48,6 +50,19 @@ func GetNetlinkProvider() NetlinkProvider {
 	return netlinkProvider
 }
 
+// HasRdmaParam returns true if PCI device has "enable_rdma" param
+// equivalent to "devlink dev param show pci/0000:d8:01.1 name enable_rdma"
+func (defaultNetlinkProvider) HasRdmaParam(pciAddr string) (bool, error) {
+	param, err := nl.DevlinkGetDeviceParamByName("pci", pciAddr, "enable_rdma")
```

Comment:
this will not work for devices on auxiliary bus. (in case of SFs which are supported by device plugin)

Reply:
Fixed